sdk/python/setup.py

```diff
@@ -52,7 +52,7 @@
     "mmh3",
     "pandas>=1.0.0",
     "pandavro==1.5.*",
-    "protobuf>=3.10",
+    "protobuf>=3.10,<3.20",
```

Comment:
nit: do we want to also update `py3.7-requirements.txt` and other files?

Reply:
oh yeah good point

Reply:
just read through what the `make lock-python-*dependencies` command does and existing requirements file - I don't think any changes are needed here.